amqp_influxdb/__init__.py

```diff
@@ -44,6 +45,24 @@ def __init__(self,
 
         connection_parameters = connection_parameters or {}
 
+        if isinstance(connection_parameters.get('credentials'), dict):
+            credentials = connection_parameters['credentials']
+            credentials_object = pika.credentials.PlainCredentials(
+                username=credentials['username'],
+                password=credentials['password'],
+            )
+            connection_parameters['credentials'] = credentials_object
+
+        if connection_parameters['ca_path'] != '':
+            connection_parameters['ssl'] = True
```

Comment:
UX-wise, I think it will be help if we have the SSL mode (true/false) setting explicit, instead of deducting it from the existence of ca_path. That also means that if it was set and ca_path is empty - an exception should be raised.